### ISSUE TITLE
the code cannot be seen

### ISSUE THREAD

**Issue by aviadr1** [googlecolab/colabtools#2617]
**Describe the current behavior**
[long time user of colab, this started happening to me today across all notebooks]
When I unhide a code section, I can see the code for a split second, but afterwards the code disappears.
the code is still there, tough, it can be copy-pasted to notebook or other tools.
but it cannot be seen. perhaps it is displayed as black text on black background?

**Describe the expected behavior**
I used to see color-coded python code in code cells.

**What web browser you are using**
Latest chrome on microsoft windows 11.

**Additional context**
screenshot:
https://www.awesomescreenshot.com/image/22120182?key=5e226a76261b4430375388882a513d98

notebook (but the issue happens to me with ANY notebook)
https://colab.research.google.com/drive/12nPd6ntCc6aSw4Po4WuHrIleprMIl0PR?usp=sharing

**Comment by cperry-goog** [comment]
We can't reproduce this, can you attempt to reproduce in an incognito window? It may be related to a browser extension.

**Comment by aviadr1** [comment]
yes I was able to reproduce it on my machine with with chrome in incognito mode.
works fine on my machine in microsoft edge
what other information would be useful?

I use colab most days, and chrome is my browser
![image](https://user-images.githubusercontent.com/6078554/154777380-c6261b25-7611-48ac-9e9a-2b4d90073cc4.png)
